src/structures/Player.ts

```diff
@@ -10,6 +10,9 @@ function check(options: PlayerOptions) {
     throw new TypeError(
       'Player option "guild" must be present and be a non-empty string.'
     );
+  
+  if (options.region && !/^\d+$/.test(options.region))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
  if (options.region && !/^\d+$/.test(options.region))
```